UnoCheck/Checkups/LinuxNinjaPresenceCheckup.cs

```diff
@@ -0,0 +1,40 @@
+﻿#nullable enable
+
+using DotNetCheck.Models;
+using DotNetCheck.Solutions;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Checkups
+{
+	public class LinuxNinjaPresenceCheckup : Checkup
+	{
+		public override bool IsPlatformSupported(Platform platform)
+			=> platform == Platform.Linux;
+
+		public override string Id => "linuxninja";
+
+		public override string Title => "Linux Ninja Build";
+
+		public override TargetPlatform GetApplicableTargets(Manifest.Manifest manifest) => TargetPlatform.SkiaGtk;
+
+		public override async Task<DiagnosticResult> Examine(SharedState history)
+		{
+			var ouput = ShellProcessRunner.Run("ninja", "--version").GetOutput().Trim();
+
+			if (ouput.Contains("not found") || ouput.Contains("No such file or directory") || ouput.Contains("ninja: command not found"))
+			{
+				return await Task.FromResult(new DiagnosticResult(
+				Status.Error,
+				this,
+				new Suggestion("Install Ninja Build System",
```

Comment:
We should redirect to the documentation for installing ninja, as it may not be the same command for all distributions. We can probably keep the `apt` if the distribution is debian based, but for others we'll need to use a link to the doc.

Reply:
@jeromelaban , looks like to install Ninja Build on Debian we need to use the same commands:

![image](https://user-images.githubusercontent.com/5515535/185985569-a12dda3b-afdd-4eb4-8812-7fb6fb7127f8.png)

Can we preserve the original solution with the check on ExitCode, instead of texts?

Reply:
Yes, debian and ubuntu use the same `apt` commands, which is fine, but other distros may not use `apt` so we need to detect that apt can be used.

As for the ExitCode, yes, use that in all cases.